content/en/docs/getting-started/_index.md

```diff
@@ -74,25 +82,40 @@ sudo systemctl enable microshift --now
 {{< /tabs >}}
 <br/>
 
-To access the cluster, install the OpenShift client or kubectl if you haven't done so yet:
+### Install Clients
+
+To access the cluster, install the OpenShift client or kubectl if you haven't done so yet.
 
 ```Bash
 curl -o oc.tar.gz https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz
 tar -xzvf oc.tar.gz
 sudo install -t /usr/local/bin {kubectl,oc}
 ```
 
-Depending on the user that will be administrating the system it may be required to copy the kubeconfig to a location that can be accessed by the user.
+### Copy or Set Kubeconfig
+
+For regular users, copy the kubeconfig to a location that can be accessed without administrator privilege.
 
 ```Bash
 mkdir ~/.kube
 sudo cp /var/lib/microshift/resources/kubeadmin/kubeconfig ~/.kube/config
 sudo chown `whoami`: ~/.kube/config
 ```
 
+Or, if running as a privileged user, export the kubeconfig.
+
+```Bash
+export KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig
+```
+
 It is now possible to run kubectl or oc commands against the MicroShift environment.
+Verify that MicroShift is running:
 
-## Using MicroShift for Application Development
+```sh
+oc get pods -A
+```
+
+## Using MicroShift All-In-One for Application Development
```

Comment:
I'm not super convinced of the "all-in-one" name, as it's not self-explanatory (what is "all"? we include CRI-O and clients, but not the component images) and therefore minimised its use. I don't have a strong preference for this particular case, though, so totally fine if we want to be more explicit here.

Reply:
`MicroShift Bundled` ? 
I feel it needs a name to be able to refer to it. 
I prefer `All-In-One` though.

Reply:
Let's leave it AIO.